nf_core/pipeline-template/{{cookiecutter.name_noslash}}/Dockerfile

```diff
@@ -5,3 +5,4 @@ LABEL authors="{{ cookiecutter.author }}" \
 COPY environment.yml /
 RUN conda env create -f /environment.yml && conda clean -a
 ENV PATH /opt/conda/envs/{{ cookiecutter.name_noslash }}-{{ cookiecutter.version }}/bin:$PATH
+RUN conda env export --name {{ cookiecutter.name_noslash }}-{{ cookiecutter.version }} > {{ cookiecutter.name_noslash }}-{{ cookiecutter.version }}.yml
```

Comment:
Tested this with the `atacseq` and `chipseq` Singularity images and it seems to be working fine.